src/backend_module.erl

```diff
@@ -101,24 +101,28 @@ generate_fun(BaseModule, RealBackendModule, F, A, TrackedFuns) ->
 fun_header(F, Args) ->
     [atom_to_list(F), "(", Args, ")"].
 
--define(METRIC(Module, Op), [backends, Module, Op]).
+time_metric(Module, Op) -> [backends, Module, Op].
+count_metric(Module, Op) -> [backends, Module, Op, count].
 
 generate_fun_body(false, _, RealBackendModule, F, Args) ->
     ["    ", RealBackendModule, ":", fun_header(F, Args), ".\n"];
 generate_fun_body(true, BaseModule, RealBackendModule, F, Args) ->
     FS = atom_to_list(F),
     %%     returned is the following
+    %%     mongoose_metrics:update(global, count_metric(Backend, F), 1),
     %%     {Time, Result} = timer:tc(Backend, F, Args),
-    %%     mongoose_metrics:update(global, ?METRIC(Backend, F), Time),
+    %%     mongoose_metrics:update(global, time_metric(Backend, F), Time),
     %%     Result.
-    ["    {Time, Result} = timer:tc(", RealBackendModule, ", ", FS, ", [", Args, "]), \n",
-     "    mongoose_metrics:update(global, ",
-     io_lib:format("~p", [?METRIC(BaseModule, F)]),
-     ", Time), \n",
+    CountMetric = io_lib:format("~p", [count_metric(BaseModule, F)]),
+    TimeMetric = io_lib:format("~p", [time_metric(BaseModule, F)]),
+    ["    mongoose_metrics:update(global, ", CountMetric, ", 1), \n",
```

Comment:
What are whitespaces for?

Reply:
Both a the beginning `["    mongoose_metrics:`... and at the end, before new line character   ...`1), \n",`